golf2/apps/aes_test/aes.c

```diff
@@ -1,8 +1,18 @@
 #include <stdio.h>
 
-#include "aes.h"
+#include "gaes.h"
 #include "tock.h"
 
+int bytes_encrypted = 0;
+
+static void tock_aes_encrypt_done_cb(int count,
+				    int unused2 __attribute__((unused)),
```

Comment:
This is something I find annoying about rust. So you can't use _ as a name in the formal parameters to mean "not used"? You need all this glop?

Reply:
This is C code, not Rust. In Rust, you can use `_arg_name` to tell the compiler to suppress unused warnings for that argument.